glottolog3/scripts/bp/diff_check.py

```diff
@@ -0,0 +1,90 @@
+#Pre-development Notes:
+#scan through identifier for all iso639-3 code
+#make the csv to a hashtable then for glottolog data only needs o(1) retrieval time, if error, means not exist
+#arr1 = what wikitounge has and glottolog doesnt, save the iso code
+#for special iso code wikitounge make, see if name exists in db, if yes find corresponding glottolog code, else indicate that as new lang
+#arr2 = what glottolog has and wikitounge doesnt then save those iso code and find corresponding glottolog code by saving that identifier id
+
+from sqlalchemy import create_engine  
+from sqlalchemy import Column, String  
+from sqlalchemy.ext.declarative import declarative_base  
+from sqlalchemy.orm import sessionmaker
+import csv, sqlite3
+
+#change this db_string to the actual postgres db containing the data
+db_string = "postgres://postgres:@127.0.0.1:5432/glottolog-3.2";
```

Comment:
I think you can pull this string from a config file, so this doesn't break if the db_string ever changes.